README.md

```diff
@@ -221,6 +222,8 @@ Default: component namespace path + `/**` (e.g. `my/app/**`)
 Glob pattern(s) for files that should be included into the preload.  
 Patterns are based on all available resources (see [resources](#resources)).
 
+**Note:** You can exclude files by prepending the exclamation mark, like so: ```!my/app/ignore.js```.
+
 ### libraries
 
 See [components](#components). Auto mode is looking for `library.js` files.
```

Comment:
I think we should also add a note here. As the current default pattern does only include `**/*.fragment.json` files but not `library-preload.json` we don't really need a equivalent exclude for libraries.